build/shade/_k-standard-goals.shade

```diff
@@ -37,9 +38,12 @@ default PACKAGELIST_JSON_FILENAME = 'NuGetPackageVerifier.json'
 }
 
 #restore-npm-modules
-  -// Find all dirs that contain a package.json file
-  var npmDirs = '${GetDirectoriesContaining(Directory.GetCurrentDirectory(), "package.json")}'
-  npm npmCommand='install ${E("KOREBUILD_NPM_INSTALL_OPTIONS")}' each='var npmDir in npmDirs'
+  @{
+    // Find all dirs that contain a package.json file
+    var npmDirs = GetDirectoriesContaining(Directory.GetCurrentDirectory(), "package.json").ToArray();
+    var npmOptions = E("KOREBUILD_NPM_INSTALL_OPTIONS") ?? " --quiet";
+    Parallel.ForEach(npmDirs, npmDir => Npm("install " + npmOptions, npmDir));
```

Comment:
Does this really take so long that it's worth turning the output (when `--quiet`) into strudel?